src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -835,7 +840,6 @@ public GitClient createClient(TaskListener listener, EnvVars environment, Run<?,
 
         String gitExe = getGitExe(n, listener);
         Git git = Git.with(listener, environment).in(ws).using(gitExe);
-
```

Comment:
We have a large number of "hidden" pull requests that risk being much more difficult to evaluate if we make changes that are pure white space changes.

Would you be willing to undo this whitespace change?

Reply:
Apologies for adding it, I am not sure how this happened.